app/models/pdc_metadata/related_object.rb

```diff
@@ -20,5 +20,21 @@ def value
     def self.new_related_object(related_identifier, related_identifier_type, relation_type)
       RelatedObject.new(related_identifier: related_identifier, related_identifier_type: related_identifier_type, relation_type: relation_type)
     end
+
+    ##
```

Comment:
Is there a style guide that suggests the `##`? A one-line comment is probably enough.